src/ec/packages/ec-component-accordion2/docs/accordion2.md

```diff
@@ -26,25 +27,22 @@ npm install --save @ecl-twig/ec-component-accordion2
 
 <!-- prettier-ignore -->
 ```twig
-{% include 'path/to/accordion2.html.twig' with {  
-  items: [  
-    {  
-      id: 'accordion-example',  
-      level: 3,  
-      toggle: {  
-        label:  
-          'Delivery of last pending proposals, a common Destiny of unity, the hour of European Democracy',  
-        icon: {  
-          name: 'energy',  
-          type: 'general',  
-          path: 'static/icons.svg',  
-          size: 's',  
-        },  
-      },  
-      content:  
-        'The College of Commissioners held today the first weekly meeting of 2019 which was devoted to discussing the challenges of this new year. Commissioners used the opportunity to take stock and discuss the year ahead, including the European elections in May and other important milestones ahead.',  
-    },  
-    ...  
-  ]  
-} %}  
+{% include 'path/to/accordion2.html.twig' with {
```

Comment:
ciao @planctus! here's a little trick: don't forget to add a trailing whitespace at the end of each line in the demo code here, otherwise it will all print on 1 line (see https://deploy-preview-106--ecl-twig.netlify.com/ec/?path=/story/components-accordion2--default)

Reply:
ahhh...i noticed that, that's why :)